app/views/registrations/school_not_confirmed/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_school_profile_path)
+    %>
+    <h2 class="govuk-heading-l">Your school needs to complete registration</h2>
+    <p>You or one of your colleagues may have tried to register before and need verify your email.</p>
+    <p>This email will be used to sign in so check with your colleagues if you need access.</p>
+    <p>What you can do</p>
```

Comment:
Do we want `:` after the text?

Reply:
The content doesn't have one https://miro.com/app/board/o9J_ldVNkCY=/?moveToWidget=3074457354086350072&cot=14 but I think it would be more correct to add it. I'll speak to shani